app/policies/application_policy.rb

```diff
@@ -0,0 +1,53 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    scope.where(id: record.id).exists?
+  end
+
+  def create?
+    false
+  end
+
+  def new?
+    create?
+  end
```

Comment:
We are doing API only app anyway, so you can drop `new?` and `edit?` from here - we will not have those routes, at least not in standard ruby sense.